pyproject.toml

```diff
@@ -9,7 +9,11 @@ dynamic = ["version"]
 description = "Python library supporting UEFI EDK2 firmware development"
 readme = {file = "readme.md", content-type = "text/markdown"}
 license = {file = "LICENSE"}
-dependencies = []
+dependencies = [
+    "pyasn1==0.4.8",
+    "pyasn1-modules==0.2.8",
+    "cryptography==39.0.0",
```

Comment:
As these true dependencies (not dev dependencies) we want to be as flexible as possible with requirements. Typically we do this via `>=` rather than `==`.

Also please require 39.0.1 or greater for cryptography: see https://cryptography.io/en/latest/changelog/#v39-0-1